test/unit/sip_arrangeSpec.js

```diff
@@ -26,6 +26,9 @@ describe('SipArrange', function() {
     _$httpBackend_.when('POST', '/filesystem/copy_to_arrange', 'filepath=c291cmNl&destination=ZGVzdGluYXRpb24%3D').respond({
       'message': 'Files added to the SIP.',
     });
+    _$httpBackend_.when('POST', '/filesystem/copy_to_arrange', 'filepath%5B%5D=cGF0aDE%3D&filepath%5B%5D=cGF0aDI%3D&destination%5B%5D=ZGVzdDE%3D&destination%5B%5D=ZGVzdDI%3D').respond({
```

Comment:
$window.encodeURI doesn't encode the = in the base64 encoded values, so it fails to match.